xerparser/reader.py

```
@@ -327,7 +348,7 @@ def __init__(self, filename):
             for row in stream:
                 try:
                     if not row:  # Skip empty rows
-                        Warning("Empty row detected on row %s", stream.line_num)
+                        warnings.warn(f"Empty row detected on row {stream.line_num}")
```

Ruff (0.14.13)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)